packages/ai-jsx/src/core/conversation.tsx

```diff
@@ -111,6 +111,13 @@ export function FunctionCall({
   return `Call function ${name} with ${partial ? '(incomplete) ' : ''}${JSON.stringify(args)}`;
 }
 
+export interface FunctionResponseProps {
```

Comment:
Assuming it's not a large number of places that needs this, did you consider `AI.PropsOfComponent<typeof FunctionResponse>`?